lasp_opensearch_data_center/constructs/frontend.py

```diff
@@ -74,6 +75,52 @@ def __init__(
         # This sets the website to whatever the passed in domain name is.
         website_url = self.hosted_zone.zone_name
 
+        # Create a Cognito Identity Pool for guest/unauthenticated access - as of 10/2024 no L2 construct available
+        identity_pool = cognito.CfnIdentityPool(
+            self, "LiberaITDCWebsiteIdentityPool",
```

Comment:
Let's remove all the "names" we can and make sure the construct IDs are generic (i.e. remove "Libera")